frontend/controller/service-worker.js

```diff
@@ -194,13 +198,29 @@ sbp('sbp/selectors/register', {
           return subscription
         }).catch(e => {
           if (!(retryCount > 3) && e?.message === 'WebSocket connection is not open') {
-            sbp('okTurtles.events/once', ONLINE, () => {
-              setTimeout(() => sbp('service-worker/setup-push-subscription', (retryCount || 0) + 1), 200)
+            return new Promise((resolve, reject) => {
+              const errorTimeoutId = setTimeout(() => {
+                reject(new Error('Timed out waiting to come back online'))
+                cancel()
+              }, 600e3)
```

Comment:
I don't think users are going to wait 10 minutes to use our app.

Could you please reduce this to 7 seconds?